api_core/google/api_core/grpc_helpers.py

```diff
@@ -244,7 +250,8 @@ class ChannelStub(grpc.Channel):
         # Use a sequence of responses:
         channel_stub.GetFoo.responses = iter([
             foo_pb2.Foo(name='bar'),
-            foo_pb2.Foo(name='baz')])
+            foo_pb2.Foo(name='baz')
```

Comment:
Missing the trailing comma (but thanks for yielding to my lame nit)

Reply:
Fixed.